badge_printing/templates/kiosk_printing/print_badges.html

```diff
@@ -5,51 +5,131 @@
         window.print();
 
         // reload to print the next badge
-        setTimeout(function() {window.location = "print_badges?minor={{ minor }}";}, 9000);
+        //setTimeout(function() {window.location = "print_badges?minor={{ minor }}";}, 9000);
```

Comment:
I expect we want this uncommented -- otherwise someone will have to be sitting at the laptop, reloading the page manually.